plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/repository/ILibraryRepositoryService.java

```diff
@@ -18,29 +18,32 @@
 
 import com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryFile;
 import com.google.cloud.tools.eclipse.appengine.libraries.model.MavenCoordinates;
-import org.eclipse.jdt.core.IClasspathEntry;
+import org.apache.maven.artifact.Artifact;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.core.runtime.IProgressMonitor;
 
 /**
  * Service interface for obtaining local paths for artifacts described by {@link MavenCoordinates}
  */
 public interface ILibraryRepositoryService {
```

Comment:
It doesn't seem that anybody uses `ILibraryRepositoryService` now except for the `ILibraryClasspathContainerResolverService` implementation — not even `ServletClasspathContainer`.  Since `ILibraryRepositoryService` explicitly exposes `Artifact`, perhaps we should just do away with the interface?